query-engine/connectors/mongodb-query-connector/src/query_strings.rs

```diff
@@ -0,0 +1,472 @@
+//! query_strings provides types to build strings representing textual mongodb queries from driver
+//! types such as Document. These are used for logging / debugging purposes mainly, and are generated
+//! lazily.
+//! There is a struct fro each different type of query to generate each of them implement the
```

Comment:
s/fro/for

Reply:
Good catch! Addressed in https://github.com/prisma/prisma-engines/pull/3332/commits/116f00d85d12c43246ea8882fa7bd75d163a7e82, but will be squashed upon merging